spring-kafka/src/test/java/org/springframework/kafka/annotation/EnableKafkaIntegrationTests.java

```diff
@@ -190,6 +192,18 @@ public void testJson() throws Exception {
 	public static class Config {
 
 		@Bean
+		public static PropertyPlaceholderConfigurer ppc() {
```

Comment:
That must be `PropertySourcesPlaceholderConfigurer` and you don't need to specify properties for it.
You can just rely on the default for PP:
```
partitions = "${zero:0}"
```